spec/CourseDbSpec.js

```diff
@@ -104,4 +104,25 @@ describe("Course DB", () => {
         expect(result)
             .toBe(true);
     });
+
+    it("shall be able to return all course names", async () => {
+        await CourseDb.createCourse("SER515", 5,
+            "");
+        await CourseDb.createCourse("SER500", 5,
+            "");
+
+        const result = await CourseDb.getCourses();
+        expect(result)
+            .toEqual(["SER515", "SER500"]);
+    });
+
+    it("shall be able to return course names based on teacher id", async () => {
+        await CourseDb.createCourse("SER515", 5,
+            "123");
+
+        const result = await CourseDb.getCourses("123");
```

Comment:
Is this the same function as above? If it is getting a list of courses for the teacher id the function name should reflect that.

Reply:
I agree with the name of the function since it works for both a generic getCourses (without ID) and with the teacher ID. I changed the teacherID to use optional parameters to make this more obvious though